frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -151,24 +151,17 @@ export const RequestedChangesDisplay = ({
                 <h5 className="margin-bottom-1">Resources</h5>
                 <ul className={"margin-0"}>
                     <li>
-                        <NavLink
+                        <USLink
                             target="_blank"
-                            to={"/resources/programmers-guide"}
+                            href="/resources/programmers-guide"
                         >
                             ReportStream Programmers Guide
-                        </NavLink>
+                        </USLink>
                     </li>
                     <li>
-                        <a
-                            className={"usa-link--external"}
-                            target={"_blank"}
-                            href={
-                                "https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html"
-                            }
-                            rel="noreferrer"
-                        >
+                        <USExtLink href="https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html">
```

Comment:
May be out of the scope of this pull request, but can we add a URL constants file for all the external links?  We could also do that for internal links for programmatic routing, but that's less straightforward.

Reply:
What kind of constraints? I do plan on taking the DOM Purify bit up to the component level as cleansing external links seems to be a good practice we follow on external links. This link, for example, would have no purification on it as of now.

Reply:
Ah constants, not constraints :) I just mean something simple like:

```js
// in src/utils/urls.ts -- or wherever!
// also I'm terrible at naming variables 
export const CDC_COVID_LIVD_CODES_URL = "https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html";
```

That way we can keep track of all external URLs in one place for easier maintenance.  In my opinion, internal paths can be handled separately since they might have wildcards and interpolation, but down to discuss this too!

Reply:
#7840